wikimedia/wikipedia-ios
# Update title view and subscribe button layout based on authentication status

**Phabricator:** https://phabricator.wikimedia.org/T310275 (partially)

### Notes
This PR updates the layout of the talk page cell depending on whether the user is logged in or logged out. See the test steps below.

### Test Steps
1. While logged out, visit a talk page and confirm the subscribe button does not appear, and that the title view is displayed to the next to the disclosure button.
2. While logged in, visit a talk page and confirm the subscribe button appears next to the disclosure button, and that the title view is displayed below this row.

### Screenshots/Videos

- Logged out:
![loggedout](https://user-images.githubusercontent.com/5652327/191362112-9504f76e-099c-4231-b286-5619719b2ba1.png)

- Logged in:
![loggedin](https://user-images.githubusercontent.com/5652327/191362115-8d7f2488-1e3f-47ba-bf02-67d4c2675416.png)
